src/components/views/Matches.js

```diff
@@ -91,7 +91,7 @@ const Matches = () => {
     // for styling: animationDelay={index + 1}
     return (
         <React.Fragment>
-            <div className="containerMatchList">
+            <div className="containerMatchList" style={{marginTop: "2em"}}>
```

Comment:
why is the style here and not within the styleClass container MatchList

Reply:
was faster. changed